content/usage/README.md

```diff
@@ -0,0 +1,80 @@
+# Using MicroShift
+MicroShift operates similar to many other Kubernetes providers. This means that you can use the same tools to deploy and manage your applications.
+
+All of the standard Kubernetes management tools can be used to maintain and modify your MicroShift applications. Below we will show some examples using kubectl, kustomize, and helm to deploy and maintain applications.
+
+## Example Applications
+
+### Metal LB
+Metal LB is a load balancer that can be used to route traffic to a number of backends.
+
+Creating the Metal LB namespace and deployment.
+```
+kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.11.0/manifests/namespace.yaml
+kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.11.0/manifests/metallb.yaml
+```
+
+Once the components are available a configMap is required to define the address pool for the load balancer to use.
+
+Create the configmap.yaml file
+```
+vi configmap.yaml
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: metallb-system
+  name: config
+data:
+  config: |
+    address-pools:
+    - name: default
+      protocol: layer2
+      addresses:
+      - 192.168.1.240-192.168.1.250
+```
+
+Load the configMap into the cluster.
+```
+kubectl create -f /tmp/configmap.yaml
+```
```

Comment:
```suggestion
Create the metallb configmap:
```
kubectl create -f - <<EOF
apiVersion: v1
kind: ConfigMap
metadata:
  namespace: metallb-system
  name: config
data:
  config: |
    address-pools:
    - name: default
      protocol: layer2
      addresses:
      - 192.168.1.240-192.168.1.250
EOF
```
```

Reply:
I like it! Will fix!

Reply:
ack, Also for the other, vim we can do the same thing.